### PR TITLE
(#4871)fix(test): Fix the jvm args `-Xmx4G` of gradle test can be overwrite by the `extraJvmArgs`.

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

Fix the oom error when run the gradle test.

### Why are the changes needed?

Fix: #4871 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Manually

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -431,9 +431,9 @@ subprojects {
     reports.html.outputLocation.set(file("${rootProject.projectDir}/build/reports/"))
     val skipTests = project.hasProperty("skipTests")
     if (!skipTests) {
-      jvmArgs = listOf("-Xmx4G")
+      val extraArgs = project.property("extraJvmArgs") as List<String>
+      jvmArgs = listOf("-Xmx4G") + extraArgs
```
Comment (mchades) [inline build.gradle.kts:435]: Can you provide relevant screenshots to prove that this parameter actually works?

Comment (diqiu50) [reply build.gradle.kts:435]: WeChat627be692d957c301004fa7a165a7ec33

Comment (mchades) [reply build.gradle.kts:435]: Should this be backported to branch-0.6?

Comment (diqiu50) [reply build.gradle.kts:435]: Need backport

**Review by jerryshao — APPROVED** [review]

**Pull request merged**